djhtml/tests/test_suite.py

```diff
@@ -0,0 +1,34 @@
+import os
+import unittest
+
+from djhtml.modes import HTML
+
+
+class TestSuite(unittest.TestCase):
+    DIR = os.path.join(os.path.dirname(__file__), "suite")
+
+    def test_available_files(self):
+        """
+        Loop over all the files in the suite directory and compare the
+        expected output to the actual output.
+
+        """
+        for filename in os.listdir(self.DIR):
+            with self.subTest(filename):
```

Comment:
Could/should we swap the `with` and `if`? Now also `filename.out` is seen as a subtest.

(Not that it effectively matters, but still).